plugins/node/opentelemetry-instrumentation-ioredis/src/instrumentation.ts

```diff
@@ -34,9 +38,7 @@ const DEFAULT_CONFIG: IORedisInstrumentationConfig = {
   requireParentSpan: true,
 };
 
-export class IORedisInstrumentation extends InstrumentationBase<
-  typeof ioredisTypes
-> {
+export class IORedisInstrumentation extends InstrumentationBase<IORedisT> {
```

Comment:
This is great as is, but we discovered that adding the generics type to `InstrumentationBase<>` creates many issues and supplies low value.

It is recommended to replace it with `any` and remove the `IORedisT` type altogether.